### DISCUSSION TITLE
Updating the email domain of a school registered to receive teacher's benefits

### DISCUSSION THREAD

**Discussion by mgcnl** [community/community#113512] [GitHub Education]
### Hi everyone,

A teacher from our school is trying to avail of the teacher's benefits offered by github. The new email domain of our school is not being accepted since the two old domains are the ones associated with our school. Is there a way to update that information?

**Comment by samus-aran** [comment]
Hi @mgcnl 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [My selected school states I must have a specific school-issued email address but it’s incorrect, what should I do?](https://github.com/orgs/community/discussions/111352#selected-school-incorrect-details)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍

**Reply by mgcnl** [reply]
This looks promising. I will try it and will update this thread how it goes. Thank you for your help!
